### ISSUE TITLE
Allow repositioning the sort inventory button

### ISSUE THREAD

**Issue by UpcraftLP** [kyrptonaught/Inventory-Sorter#41]
When playing with the [backslot mod](https://www.curseforge.com/minecraft/mc-mods/backslot "CurseForge/backslot"), one of the slots interferes with the sort button (putting items in and out of that slot will trigger an inventory sort). Would it be possible to allow repositioning the button via config?

**Comment by kyrptonaught** [comment]
It would be possible yes, but the sort button position is dynamic as it depends on the inventory. Should I add say an offset to move the button by x and y in every inv? Or just add the option to lock it to a set x/y?

**Comment by UpcraftLP** [comment]
good point.. I guess the most sensible approach would be to let users set an absolute position in the config, but have the option to override that per screen (could use the screen handler's registry ID)?
